### ISSUE TITLE
Allow `wx:` directives in `<template is>` and `<include>`

### ISSUE THREAD

**Issue by LastLeaf** [wechat-miniprogram/glass-easel#30]
These are supported in legacy framework.

**Comment by LastLeaf** [comment]
Also for `<slot>`

**Comment by SgLy** [comment]
slot with `wx:else` directive may lost its name, reproducing:

```rust
const SRC: &str = r#"<block wx:if="{{true}}"></block><slot wx:else name="test"/>"#;
const GEN: &str = r#"<block wx:if="{{true}}"></block><block wx:else><slot name="test"></slot></block>"#;
let tree = parse_tmpl(SRC).unwrap();
assert_eq!(tree.to_string(), GEN);
```

**Comment by LastLeaf** [comment]
Fixed in 0fc9c22
